interfaces/apparmor/spec_test.go

```diff
@@ -280,3 +280,30 @@ func (s *specSuite) TestApparmorSnippetsFromLayout(c *C) {
 	c.Assert(updateNS[3], Equals, profile3)
 	c.Assert(updateNS, DeepEquals, []string{profile0, profile1, profile2, profile3})
 }
+
+func (s *specSuite) TestChopTree(c *C) {
```

Comment:
can we also have tests for non-absolute paths, tests for paths with double slashes, tests for the empty path, tests for negative `assumedPrefixDepth`, and tests for a deeper `assumedPrefixDepth` than the path provided?

Reply:
Yes, thought the code does run on top of cleaned paths so I could perhaps make it clean the path once again, to make sure it's sane.